### PR TITLE
Adding Weblate PR owners to the notifications

### PR DESCRIPTION
**Pull request by ctlaltdieliet**
#### Summary
Adding the Weblate PR owners to the notifications that are sent to the Mattermost i18n-channel when there is a lock in a Weblate project.

Making the double_languages.py scripts some less verbose while running
@cwarnermm

### CODE REVIEW

File: scripts/weblate/double_languages.py [hunk 1]
```diff
@@ -60,7 +60,7 @@
     except Exception as err:
         print('Other error occurred while notifying Mattermost channel: '+err.message+' '+err.args)
     try:
-        responseWeblate=w.request('post','https://translate.mattermost.com/api/components/i18n-wip/'+projects[project]['wip']+'/lock/',{'lock':True})
+        responseWeblate=w.request('post','https://translate.mattermost.com/api/components/i18n-wip/'+projects[project]['wip']+'/lock/',{'lock':True
```
Comment (neflyte) [inline scripts/weblate/double_languages.py]: I think the last two characters of this statement (`})`) were cut off. You might want to double-check.

File: scripts/weblate/weblate_locks_notify_community.py [hunk 2]
```diff
@@ -17,18 +17,28 @@
...
+mention['server']='cc: @guillermo.vaya,@zef.hemel'
+mention['webapp']='cc: @guillermo.vaya,@zef.hemel'
+mention['focalboard']='cc: @jesus.espino,@scott.bishel'
+mention['playbooks']='cc: @jesse.hallam,@caleb.roseland'
+mention['glossary']=''
+
+w=wlc.Weblate(key=apikey.myAPIKey)
 current_locks={}
-previous_locks= json.load(open("locks.txt"))
+previous_locks= json.load(open("/home/tom/scripts/mattermost/locks.txt"))
```
Comment (neflyte) [inline scripts/weblate/weblate_locks_notify_community.py]: Will the `/home/tom/scripts/mattermost` path exist on the Weblate server? If not, this path may need to be modified.

File: scripts/weblate/weblate_locks_notify_community.py [hunk 3]
```diff
@@ -40,4 +50,4 @@
         print(f'Other error occurred while notifying Mattermost-channel: {err.message} {err.args}')
     else:
         print('Success!')
-json.dump(current_locks,open("locks.txt",'w'))
\ No newline at end of file
+json.dump(current_locks,open("/home/tom/scripts/mattermost/locks.txt",'w'))
```
Comment (neflyte) [inline scripts/weblate/weblate_locks_notify_community.py]: Same concern as previous: will the `/home/tom/scripts/mattermost` path exist on the Weblate server? If not, this path may need to change.

**Review by neflyte — CHANGES_REQUESTED** [review]

File: scripts/weblate/double_languages.py (hunk 1 above)
Comment (ctlaltdieliet) [reply scripts/weblate/double_languages.py]: mmm, strange, how did that happen...
But solved it

File: scripts/weblate/weblate_locks_notify_community.py (hunk 2 above)
Comment (ctlaltdieliet) [reply scripts/weblate/weblate_locks_notify_community.py]: yeah, mixing two branches, one that is running at my local server and official one :facepalm: 
Also working on the train in a terminal with vi was not my best idea

File: scripts/weblate/weblate_locks_notify_community.py [hunk 4]
```diff
@@ -17,18 +17,28 @@
 projects['playbooks']='playbooks/webapp'
 projects['glossary']='mattermost/glossary'
 
-w=wlc.Weblate(apikey.myAPIKey)
+mention={}
+mention['mobile']='cc: @guillermo.vaya,@zef.hemel'
+mention['server']='cc: @guillermo.vaya,@zef.hemel'
+mention['webapp']='cc: @guillermo.vaya,@zef.hemel'
+mention['focalboard']='cc: @jesus.espino,@scott.bishel'
+mention['playbooks']='cc: @jesse.hallam,@caleb.roseland'
```
Comment (cwarnermm) [inline scripts/weblate/weblate_locks_notify_community.py:26]: One more Dev owner can be added: Desktop App issues are owned by @devinbinnie

Comment (ctlaltdieliet) [reply scripts/weblate/weblate_locks_notify_community.py:26]: added!

**Review by cwarnermm — APPROVED** [review]
Thank you so much, @ctlaltdieliet!

**Pull request merged**